packages/truffle/scripts/test.sh

```diff
@@ -3,7 +3,7 @@
 set -o errexit
 
 if [ "$GETH" == true ]; then
-  npm run build-cli && mocha --timeout 50000 --grep @ganache --invert --colors
+  npm run build-cli && mocha --timeout 50000 --grep @ganache --invert --colors $@
 else
-  npm run build-cli && mocha --no-warnings --timeout 7000 --grep @geth --invert --colors
+  npm run build-cli && mocha --no-warnings --timeout 7000 --grep @geth --invert --colors $@
```

Comment:
What's being added here?

Reply:
It forwards arguments; it's just for doing `yarn test -b` so that the `-b` gets forwarded to mocha.